.github/workflows/trivy_scan.yml

```diff
@@ -0,0 +1,29 @@
+name: Trivy vulnerability scan
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 4 * * *' # Nightly 4AM UTC
```

Comment:
we have decided to add this to every workflow file:

```
defaults:
  run:
    # fail-fast using bash -eo pipefail. See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference
    shell: bash
```

but given that this doesn't contain any scripts it feels OK to merge as-is. Any thoughts @michalnicp?